cvmfs/cvmfs_server

```diff
@@ -2026,6 +2027,38 @@ Alias /cvmfs/${name} ${storage_dir}
 EOF
 }
 
+create_apache_config_for_global_info() {
+  local conf_file="cvmfs.info.conf"
+
+  ! has_apache_config_file $conf_file || return 0
+
+  # TODO(rmeusel): make this a bit more flexible
+  local storage_dir="${DEFAULT_LOCAL_STORAGE}/info"
+
+  # TODO(rmeusel): fine-tune this configuration file
+  create_apache_config_file $conf_file << EOF
+# Created by cvmfs_server.  Don't touch.
+
+KeepAlive On
+Alias /cvmfs/info ${storage_dir}
+<Directory "${storage_dir}">
+    Options -MultiViews
+    AllowOverride Limit
+    $(get_compatible_apache_allow_from_all_config)
+
+    EnableMMAP Off
+    EnableSendFile Off
+
+    FileETag None
+
+    ExpiresActive On
+    ExpiresDefault "access plus 3 days"
+    ExpiresByType text/html "access plus 5 minutes"
+    ExpiresByType application/x-cvmfs "access plus 2 minutes"
+</Directory>
+EOF
+}
+
```

Comment:
I've taken your `create_apache_config_core()` as an inspiration to actually [DRY up the config creation](https://github.com/reneme/cvmfs/commit/b334daa56cafd1f0f13832ae5a33e927336c9f84) of `mkfs` and `add-replica` which was indeed a code duplication. However, I think it makes sense to keep the `/cvmfs/info` config separate and optimise it for its purpose, no?